client/interfaces.go

```diff
@@ -228,7 +228,7 @@ type InstanceServer interface {
 	DeleteNetwork(name string) (err error)
 
 	// Network ACL functions ("network_acl" API extension)
-	GetNetworkACLs() (acls []api.NetworkACL, err error)
+	GetNetworkACLs() (acl []api.NetworkACL, err error)
```

Comment:
Why this change? `acls` sounds correct given it's a list, also wrong commit :)

Reply:
Agreed, fixed.